sql/firefox_desktop_exact_mau28_by_dimensions_v1.sql

```diff
@@ -4,10 +4,7 @@ SELECT
   COUNT(*) AS mau,
   COUNTIF(last_seen_date = submission_date) AS dau,
   -- requested fields from bug 1525689
-  source,
-  medium,
-  campaign,
-  content,
+  attribution.*,
```

Comment:
This is neat, though a bit magical. I'd prefer to be more explicit and list out all four fields individually as `attribution.source`, etc. unless that requires additional aliasing.

Alternately, is it possible to `group by attribution` and use the struct as-is? I expect we may want to add additional dimensions in a future version, and it may be nice to continue to have these fields grouped in a struct.

Reply:
we can't group by the struct, but I can list out the fields explicitly